include/flexflow/ops/concat.h

```diff
@@ -14,11 +27,19 @@ class ConcatMeta : public OpMeta {
 
 class Concat : public Op {
 public:
+  using Params = ConcatParams;
+  using Input = std::vector<ParallelTensor>;
+
   Concat(FFModel& model,
          int n,
          const ParallelTensor* inputs,
          int axis,
          const char* name);
+  Concat(FFModel& model,
+         const ConcatParams& params,
+         const std::vector<ParallelTensor>& inputs,
+         bool allocate_weights,
```

Comment:
I don't think `Concat` has any weights, so I think this parameter should be omitted (I understand why it was added, see my previous comment on what I think we should do in general about this).